### ISSUE TITLE
How to use DeepSHAP with keras Sequential models?

### ISSUE THREAD

**Issue by ahmedabbas81** [shap/shap#825]
Dear all,

Can you please give me an example on how to use DeepSHAP on keras Sequential model? I already trained the keras model, and I need to know how much does each input contributes to each output. I tried to use the example given in the readme, but it gave me all the contibutions are zeros which is for sure wrong.

This is the model code:

model = Sequential()
                                model.add(Dense(first_layer[i], activation=act_fn[k], input_dim=X_train.shape[1]))
                                model.add(Dense(second_layer[j], activation=act_fn[k]))
                                model.add(Dense(y_train.shape[1], activation='sigmoid'))
                                model.compile(optimizer=opt[l],
                                              loss=losses[m],
                                              metrics=['accuracy'])
                                model.fit(X_train,y_train,batch_size = batch_size1[o], epochs=epochs1[n])

Thanks

**Comment by tqfjo** [comment]
Similar issue with full repro: https://github.com/slundberg/shap/issues/942
